### PR TITLE
Update base-nl.yaml

### PR DESCRIPTION
**Pull request by Underlyingglitch**
Full translation to dutch.

### CODE REVIEW

File: translations/base-nl.yaml [hunk 1]
```diff
@@ -2,51 +2,49 @@ steamPage:
...
-        And as if that wasn't enough, you also have to produce exponentially more to satisfy the demands - the only thing that helps is scaling!
+        En als dat nog niet genoeg is moet je ook nog eens steeds me produceren om aan de vraag te kunnen voldoen. Het enige dat helpt is uitbreiden!
 
-        While you only process shapes at the beginning, you have to color them later - for this you have to extract and mix colors!
+        Ondanks het feit dat je in het begin alleen vormen maakt, komt er het punt waarop je ze moet kleuren. Deze kleuren moet je vinden en mengen!
 
-        De game kopen op steam geeft je toegang tot de volledige versie, maar je kan ook de demo spelen op shapez.io en later beslissen!
-    title_advantages: Standalone Voordelen
+        Door het spel op Steam te kopen kun je de volledige versie spelen. Je kunt echter ook een demo versie spelen op shapez.io en later beslissen.
+    title_advantages: Losstaande Voordelen
```
Comment (daanbreur) [inline translations/base-nl.yaml:16]: leave this `Standalone`

File: translations/base-nl.yaml [hunk 2]
```diff
@@ -2,51 +2,49 @@ steamPage:
...
+        - Ongelimiteerde Markers
+        - Help mij! ❤️
+    title_future: Geplande Content
     planned:
-        - Blauwprint Bibliotheek (Standalone Exclusief)
-        - Steam Achievements
-        - Puzzel Modus
+        - Blueprint Bibliotheek (Alleen volledige versie)
+        - Steam Prestaties
+        - Puzzle Modus
```
Comment (daanbreur) [inline translations/base-nl.yaml:30]: `puzzle` is english
`puzzel` is dutch

File: translations/base-nl.yaml [hunk 3]
```diff
@@ -2,51 +2,49 @@ steamPage:
...
         - Minimap
         - Mods
         - Sandbox modus
         - ... en nog veel meer!
     title_open_source: Dit spel is open source!
     title_links: Links
     links:
-        discord: Officiële Discord
+        discord: Officiele Discord
```
Comment (daanbreur) [inline translations/base-nl.yaml:38]: You write it with the 2 dots. So don't Remove them

File: translations/base-nl.yaml [hunk 4]
```diff
@@ -346,39 +341,39 @@ ingame:
     connectedMiners:
         one_miner: 1 Miner
         n_miners: <amount> Miners
-        limited_items: Limited to <max_throughput>
+        limited_items: Limiet ingesteld op: <max_throughput>
```
Comment (daanbreur) [inline translations/base-nl.yaml:344]: This must be `Gelimiteerd op`

**Review by daanbreur — CHANGES_REQUESTED** [review]

**Comment by RemcoStamHub** [comment]
Can confirm all comments of daanbreur are correct.

**Comment by Underlyingglitch** [comment]
@daanbreur You made some very good comments. Indeed thy are correct. Sorry I missed those. Now, I’m new to github pull requests: do I have to change this? If so please let me know,I would be happy to do so!

**Comment by daanbreur** [comment]
Go to your local fork. `github.com/<username>/shapez.io`

Then change the branch from `master` to `patch-1`. ( there is a little selector at the top of the page) 

Then just navigate to the Dutch translation file. And make the edits. All commits on that branch (patch-1) will be added to this PR.

File: translations/base-nl.yaml (hunk 4 above)
Comment (daanbreur) [reply translations/base-nl.yaml:344]: Or `Gelimiteerd tot`

**Comment by tobspr** [comment]
I'll make the adjustments - thanks a lot to all of you!

**Comment by daanbreur** [comment]
Thanks...

**Pull request merged**